### PR TITLE
Create gerbradley-belfast.md

### PR DESCRIPTION
**Pull request by gerbradley**
Added some info about Belfast

### CODE REVIEW

File: gerbradley-belfast.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Places to Eat
+ - Howard St
+ - The John Hewitt
+ - Muriels
+
+### Things to Do
+ - Eat
+ - Drink 
+ - Sleep
```
Comment (noelvaneynde-exclaim) [inline gerbradley-belfast.md:9]: Sleeping is good

File: gerbradley-belfast.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### Places to Eat
+ - Howard St
+ - The John Hewitt
+ - Muriels
```
Comment (noelvaneynde-exclaim) [inline gerbradley-belfast.md:4]: What kind of food do they serve?

File: gerbradley-belfast.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+### Places to Eat
+ - Howard St
+ - The John Hewitt
```
Comment (JohnDeans) [inline gerbradley-belfast.md:3]: @gerbradley A pint sounds good

File: gerbradley-belfast.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+ - Muriels
+
+### Things to Do
+ - Eat
+ - Drink
```
Comment (ravno) [inline gerbradley-belfast.md:8]: @gerbradley At what place would you like to drink?

**Pull request merged**